sdk/create-dapp/src/index.ts

```diff
@@ -6,43 +6,57 @@
 import { existsSync, statSync } from 'fs';
 import { mkdir, readdir, readFile, writeFile } from 'fs/promises';
 import { relative, resolve } from 'path';
+import { parseArgs } from 'util';
 import { prompt } from 'enquirer';
 
+const { values: args } = parseArgs({
+	options: {
+		template: {
+			type: 'string',
+			default: '',
+			short: 't',
+		},
+	},
+});
+
 async function main() {
 	const results = await prompt<{
 		template: string;
 		dAppName: string;
-	}>([
-		{
-			type: 'select',
-			name: 'template',
-			message: 'Which started template would you like to use?',
-			choices: [
-				{
-					name: 'react-client-dapp',
-					hint: 'React Client dApp that reads data from wallet and the blockchain',
-				},
-				{
-					name: 'react-e2e-counter',
-					hint: 'React dApp with a move smart contract that implements a distributed counter',
-				},
-			],
-		},
-		{
-			type: 'input',
-			name: 'dAppName',
-			message: 'What is the name of your dApp? (this will be used as the directory name)',
-			initial: 'my-first-sui-dapp',
-		},
-	]);
+	}>(
+		[
+			{
+				type: 'select',
+				name: 'template',
+				message: 'Which started template would you like to use?',
+				choices: [
+					{
+						name: 'react-client-dapp',
+						hint: 'React Client dApp that reads data from wallet and the blockchain',
+					},
+					{
+						name: 'react-e2e-counter',
+						hint: 'React dApp with a move smart contract that implements a distributed counter',
```

Comment:
thx for the more explicit descriptions from the previous PR, this makes it a lot more obvious :)